lib/src/components/NxSmallTag/NxSmallTag.scss

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use '../../scss-shared/nx-text-helpers';
+
+.nx-small-tag {
+  @include nx-text-helpers.semi-bold;
+
+  align-items: center;
+  border-radius: 8px;
+  box-sizing: border-box;
+  color: var(--nx-swatch-white);
+  display: inline-flex;
+  font-size: var(--nx-font-size-s);
+  height: 24px;
+  margin: 0 var(--nx-spacing-2x);
+  max-width: 200px;
+  padding: 0 var(--nx-spacing-2x);
+  position: relative;
+  vertical-align: middle;
+
+  + .nx-small-tag {
+    margin-left: 0;
+  }
+}
+
+.nx-small-tag-color {
```

Comment:
This shouldn't be a separate base class.  The color classes should be `nx-small-tag--blue` etc.

Reply:
Updated here 9eda9c3227c6741d2f3bbfd27aa1af289d5fcd59